packtools/sps/validation/footnotes.py

```diff
@@ -80,67 +81,171 @@ def fn_validation(self):
         if not dtd:
             return
 
-        fns = ArticleFootnotes(self.xmltree)
-        for fn in fns.article_footnotes:
+        for fn in self.fns_dict.get("fns"):
```

Comment:
@Rossi-Luciano uso excessivo de for, sendo que há a classe FootnoteValidation e FootnotesValidation.

Na FootnoteValidation (singular), crie todas as validações SEM for, é para ser aplicado em um item do `self.fns_dict.get("fns")`

Na FootnotesValidation (plural), faça 1 for

Reply:
OK, @robertatakenaka